classes/02class/exercises/c02-network08/starkmatt/c02-network08.txt

```diff
@@ -0,0 +1,17 @@
+##Exercise 8
+What is the difference between inbound x outbound?
+- Inbound rules ares used to filter traffic coming from the internet to the local network. You'll be able to add rule sets to specific ports and IP addresses if they're required.
+- Outbound rules are used to filter traffic coming from the local network out to the internet.
+
+What is going to happen by default? Everything allowed or denied?
+By default, Everythingis allowed as the Source is 0.0.0.0/0 
```

Comment:
Can you detail it? 

What's allowed?  Internal traffic, internet traffic? Inbound or outbound?